### ISSUE TITLE
Overview and tree view empty in shinyMixR interface.

### ISSUE THREAD

**Issue by sg033** [RichardHooijmaijers/shinyMixR#9]
Hi, I'm trying to teach myself nlmixr and also how to use shinyMixR. 

When I create a new project and then run shinymixr in a browser (run_shinymixr(launch.browser=TRUE)) , the interface appears but is empty.  I am running it on windows and have shiny, nlmixr, and shinymixr installed. It's most likely that I have done something incorrectly but can't figure out what.

Any help would be greatly appreciated. Thanks.

**Comment by RichardHooijmaijers** [comment]
I am not entirely sure but I suspect it has something to do with the working directory. I would make sure that the working directory is the location where your project (models) are located.
You can check with `getwd()` if this is correct. As a first test you can create a start project in your current working directory using `create_proj()`. Then you can try to start the interface using `run_shinymixr(launch.browser = TRUE)`. If this doesn't work could you please provide some additional information (e.g. output from `sys.Info()` and warnings/errors issued by R)?

**Comment by sg033** [comment]
My project is in"C:/ProjectFolder". When I check with getwd() it confirms
this:
> getwd()
[1] "C:/ProjectFolder"
Also, when I also create a new project in the working directory as you said
and start the interface, it is still empty.

The only warning messages I get are:
1: package ‘shiny’ was built under R version 3.6.3
2: package ‘nlmixr’ was built under R version 3.6.3

Sys.info
sysname: "Windows"  release: "10 x64"      version  "build 15063"
machine "x86-64"

If you need any further information, just let me know. Thanks so much.

**Comment by RichardHooijmaijers** [comment]
I did some additional testing and initially had similar issues. After updating packages everything worked as expected. I would at least install the latest development version of shinyMixR `devtools::install_github("richardhooijmaijers/shinyMixR")`. This will also ask to install/update all related packages. If you have done this and still encounter issues could you provide the output from  `sessionInfo()` ? This will give me a chance to look at all the package versions and try to reproduce the behavior.

**Comment by sg033** [comment]
I tried to update the version of shinyMixR in R 3.6.1 that came with the
windows installer for nlmixr but it didn't work, I got the error:

Failed to install 'shinyMixR' from GitHub:

  (converted from warning) installation of package
‘C:/Users/AppData/Local/Temp/Rtmp8Ipr6l/file4dec1884734/shinyMixR_0.2.0.tar.gz’
had non-zero exit status


  So then I decided to download R 4.0 and then installed all of the
programmes again, including shinymixr, and it worked! Wonderful. Thank you
so much for all your help.
